web/src/views/Compare.vue

```diff
@@ -1,9 +1,9 @@
 <template>
   <v-container fluid fill-height>
     <v-row justify="center">
-      <v-col cols="12">
+      <v-col cols="12" class="no-y-padding">
         <CompareCard
-          v-if="diff"
+          v-if="diff && diff.blocks"
```

Comment:
```suggestion
          v-if="diff?.blocks"
```
If typescript works here.

Reply:
I'm afraid the template loader cannot handle typescript no.